SPECS/ksh/ksh.spec

```diff
@@ -1,23 +1,16 @@
-%global       verBetaPrefix 1.0.0
-%global       verBetaSuffix 1
-%global       verBetaFull %{verBetaPrefix}-beta.%{verBetaSuffix}
-
+%global debug_package %{nil}
```

Comment:
This looks fishy. Why do we want to disable the debuginfo package now?

Reply:
I added this line to fix the build failure:

In SRPM:
time="2024-02-01T08:51:28Z" level=debug msg="Processing files: ksh-debuginfo-1.0.8-1.azl3.x86_64"
time="2024-02-01T08:51:28Z" level=debug msg="error: Empty %files file /usr/src/mariner/BUILD/ksh-1.0.8/debugfiles.list"

In test:
time="2024-02-01T08:51:32Z" level=debug msg="RPM build errors:"
time="2024-02-01T08:51:32Z" level=debug msg="error: Empty %files file /usr/src/mariner/BUILD/ksh-1.0.8/debugfiles.list"

Reply:
Got it. Could you in that case add include that in a comment above this line for future refernce?